### DISCUSSION TITLE
How to customize logo?

### DISCUSSION THREAD

**Discussion by somdeepkundu** [giswqs/geog-414#4] [Q&A] [answered]
Thank you, professor.
Question-How to create logo url and customize size?

I have tried online image to url converter, but it's not giving desired results. 
![image](https://github.com/giswqs/geog-414/assets/62704009/05ae52a5-4dbb-49a7-9369-51d1ac132f36)

**Comment by giswqs** [comment, marked as answer]
It should be a direct link to an image, like https://i.imgur.com/at4Qprk.png
